### PR TITLE
Fixed help command

### PR DESCRIPTION
**Pull request by L2jLiga**
Command: `terser --help options`

*Actual:* `TypeError: UglifyJS.default_options is not a function`
*Expected:* Help message

### CODE REVIEW

**Comment by fabiosantoscode** [comment]
Thanks for the PR, nice catch!

**Comment by fabiosantoscode** [comment]
The build is failing because these files call `require` on tools/node:

```
test/node.js:module.exports = require("../tools/node");  
test/ufuzz.js:var UglifyJS = require("../tools/node.js");
```

**Comment by L2jLiga** [comment]
Thanks.
Now it looks like all is fine

File: main.js [hunk 1]
```diff
@@ -38,3 +38,4 @@ export {
     tokenizer,
 } from "./lib/parse.js";
 export { to_ascii } from "./lib/minify.js";
+export { default_options } from "./tools/node";
```
Comment (nifgraup) [inline main.js]: This should be moved to the `// CLI` section, `// TESTS` exports should be removed from the main bundle at some point.

Comment (L2jLiga) [reply main.js]: Good point, thx

Will update PR soon

**Comment by fabiosantoscode** [comment]
Thanks!

**Pull request merged**